tools/prepare-release/main.go

```diff
@@ -71,10 +71,6 @@ func main() {
 	cmd = exec.Command("mv", "changelog/unreleased", newChangelog)
 	run(cmd)
 
-	// install release-deps: calens
-	cmd = exec.Command("make", "release-deps")
-	run(cmd)
-
```

Comment:
@SagarGi "something happened" - lines like this should not be deleted in a tests-only PR.